### PR TITLE
Added committee wg names and leaders

### PR DESCRIPTION
**Pull request by fubar2**
As promised.
Promoted me to emeritus while I was at it :)

### CODE REVIEW

**Comment by fubar2** [comment]
Dunno why but the editor I use seems to remove dangling space at the end of a line - hence the one lopped at the end of Frederik's entry - probably harmless.

File: src/community/steering/index.md [hunk 1]
```diff
@@ -29,12 +29,18 @@ See the [Galaxy Governance Model](/src/community/governance/index.md) for how th
 
 ## Subcommittees
 
-Subcommittees will be added here as they are formed.
+These are working groups, similar to the technical groups. They are open to interested community members willing to add their contribution.
+Please contact the committee member leading the project if you would like to learn more about participating in their work.
+
+* Membership - Dan Blankenberg
+* Correcting misconceptions - Hans-Rudolph Hotz
```
Comment (beatrizserrano) [inline src/community/steering/index.md:36]:
```suggestion
* Correcting misconceptions - Hans-Rudolf Hotz
```

**Review by beatrizserrano — APPROVED** [review]
thank you @fubar2 :)

File: src/community/steering/index.md (hunk 1 above)
Comment (fubar2) [reply src/community/steering/index.md:36]: whoopsie - thanks for spotting that!

**Comment by tnabtaf** [comment]
Thank you @fubar2 !  Except for the part where you took out the "Thank you Ross."  :-)

**Comment by fubar2** [comment]
Thanks for merging. Self appointed Galaxy project emeritus is my new title.

**Pull request merged**